### PR TITLE
[config] Enables swagger UI by default

### PR DESCRIPTION
**Pull request by dpgaspar**
### CATEGORY

- [ ] Bug Fix
- [X] Enhancement (new features, refinement)
- [ ] Refactor
- [ ] Add tests
- [ ] Build / Development Environment
- [ ] Documentation

### SUMMARY
This is a proposal to enable FAB Swagger UI by default. To access the UI, user's need to have a specific permission `can show on SwaggerView`, if set by default `superset init` will create the permission for swagger and associate it to the `Admin` role.

### BEFORE/AFTER SCREENSHOTS OR ANIMATED GIF
image

### ADDITIONAL INFORMATION
- [ ] Has associated issue:
- [ ] Changes UI
- [ ] Requires DB Migration.
- [ ] Confirm DB Migration upgrade and downgrade tested.
- [X] Introduces new feature or API
- [ ] Removes existing feature or API

### CODE REVIEW

**Comment by willbarrett** [comment]
I'm OK with this, given that it's behind a permission.

**Review by willbarrett — APPROVED** [review]

File: superset/config.py [hunk 1]
```diff
@@ -162,7 +161,9 @@ def _try_json_readsha(filepath):
 # Uncomment to specify where clicking the logo would take the user
 # e.g. setting it to '/welcome' would take the user to '/superset/welcome'
 LOGO_TARGET_PATH = None
-
+# Enables SWAGGER UI for superset openapi spec
+# ex: http://localhost:8080/swaggerview/v1
+FAB_API_SWAGGER_UI = True
 # Druid query timezone
 # tz.tzutc() : Using utc timezone
 # tz.tzlocal() : Using local timezone
```
Comment (villebro) [inline superset/config.py]: nit: I kinda like having a blank line highlighting separate sections. Would prefer to leave the old blank line after `LOGO_TARGET_PATH` and also adding a new one after `FAB_API_SWAGGER_UI`

**Review by villebro — CHANGES_REQUESTED** [review]
I think adding a row to the changelog would be appropriate, so nobody unknowingly exposes the swagger schema by mistake. Other than that (and the nit) LGTM.

File: superset/config.py (hunk 1 above)
Comment (dpgaspar) [reply superset/config.py]: Makes sense

**Review by villebro — APPROVED** [review]
LGTM; would still like a note in the changelog, but ok without it, too.

**Pull request merged**